cmd/protoc-gen-go-grpc/go.mod

```diff
@@ -0,0 +1,8 @@
+module google.golang.org/grpc/cmd/protoc-gen-go-grpc
+
+go 1.14
```

Comment:
I know Go 1.11 is not technically supported by the Go team anymore. But if AppEngine is still a supported target, it still has a Go 1.11 runtime. And Go 1.11 does not gracefully handle these `go` version declarations.

Reply:
1.14 was inadvertent. Changed to 1.9, which is the oldest Go version supported by the protobuf module(s).

It's unfortunate that 1.9 is different than the version supported by the top-level gRPC module, but I think it's better (and possibly necessary) for this to remain in sync with the protobuf module which will import it.